wp-menu-cart.php

```diff
@@ -501,17 +571,30 @@ function add_wpmenucart_item_class ($classes) {
 			return $classes;
 		}
 		*/
+
 		$classes = apply_filters( 'wpmenucart_menu_item_classes', $classes );
 		$this->menu_items['menu']['menu_item_li_classes'] = $classes;
 
 		// DEPRECATED: These filters are now deprecated in favour of the more precise filters in the functions!
-		$wpmenucart_menu_item = apply_filters( 'wpmenucart_menu_item_filter', $this->wpmenucart_menu_item() );
-
-		$item_data = $this->shop->menu_item();
+		$menu_item_li = apply_filters_deprecated( 'wpmenucart_menu_item_filter', array( $this->wpmenucart_menu_item() ), '2.5.3', '' );
 
-		$menu_item_li = '<li class="'.$classes.'" id="wpmenucartli">' . $wpmenucart_menu_item . '</li>';
+		return '<li class="'.$classes.'" id="wpmenucartli">' . $menu_item_li . '</li>';
+	}
+	
+	/**
+	 * Add Menu Cart to menu
+	 * 
+	 * @return menu items + Menu Cart item
+	 */
+	public function add_itemcart_to_menu( $items ) {
+		$common_classes = '';
+		if ( $this->get_common_li_classes( $items ) != '' ) {
+			$common_classes .= $this->get_common_li_classes( $items );
+		}
```

Comment:
This can be simplified to `$common_classes .= $this->get_common_li_classes( $items );`